python/ray/data/tests/test_dataset_tfrecords.py

```diff
@@ -40,7 +40,7 @@ def test_read_tfrecords(ray_start_regular_shared, tmp_path):
     assert dict(df.dtypes) == {
         "int64": np.int64,
         "int64_list": object,
-        "float": np.float,
+        "float": np.float_,
```

Comment:
This came up when I was testing locally with Arrow 8.0+. `np.float` is [deprecated starting numpy 1.20.0](https://numpy.org/devdocs/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated), so I have replaced it with `np.float_`.